### ISSUE TITLE
[unofficial RFC] CSS is missing, opinions on moving govspeak CSS into govspeak.

### ISSUE THREAD

**Issue by mcgoooo** [alphagov/govspeak#102]
as a user of the govspeak gem, i wish all documentation in one place, and a living style guide embedded in the gem. 

This will make it easier to display changes, keep documentation up to date, and keep things consistent, it also makes it easier than trying to switch between static and govspeak.to develop. a lot of the code is a divergent, and it would be good to sort it. Currently there is three places the CSS lives.

The CSS is in the these.places currently
government frontend
the mainstream publisher
frontend.

**Comment by mcgoooo** [comment]
https://github.com/alphagov/static/compare/missing-govspeak?expand=1

**Comment by mcgoooo** [comment]
from slack with @36degrees and @danielroseman 

Some of the govspeak CSS is for functionality that doesn’t exist in the Govspeak gem - it only exists in Whitehall.

[11:19 AM]  
e.g. https://github.com/alphagov/static/blob/master/app/assets/stylesheets/govuk-component/govspeak/_fraction.scss - that’s what the ticks and crosses in the doc block are for
GitHub
alphagov/static
GOV.UK static files and resources
 

[11:19 AM]  
that magic comes from https://github.com/alphagov/whitehall/blob/19cd7d72de32454d532c195f35b027fa1b3ba6ac/app/helpers/govspeak_helper.rb
GitHub
alphagov/whitehall
whitehall - Rails application that publishes and renders most pages on www.gov.uk/government
 

[11:20 AM]  
AFAICT, whitehall supports a superset of what’s in alphagov/govspeak

[11:21 AM]  
Daniel Roseman That shouldn’t be the case any more, we spent a lot of time last year moving all the extra functionality into the gem (edited)

[11:21 AM]  
fractions is the one exception since it is literally only used by one single publication

[11:24 AM]  
Oliver Byford Aah, interesting. I think it felt like there was more of a disjunct, but yeah it is actually only that fraction file with a cross for govspeak.

[11:25 AM]  
It was very hard to work out what was going on when I was doing that work, I think originally more things had crosses until I got more clarity
